automation_tools/satellite6/upgrade/tasks.py

```diff
@@ -44,6 +45,7 @@
 
 
 logger = logger()
+data = {}
```

Comment:
This is not required as we are directly passing a dictinary to json. Need not to initialize it.